helping.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Helping out
+sidebar_link: true
+sidebar_sort_order: 300
+---
+
+Keeping Boston Python running requires lots of small jobs. If you take on one of these, it’s a big help to the organization. You can sign up for one of these and then move to doing something else later on. We really appreciate your help!
+
+## Considerations
+
+Time: rough estimate of how much effort it will take.
+
+One-time or on-going: how big a commitment are you making.
+
+One person, or shared job: can this task be shared among a few people?
+
+Importance: how important is it that it get done? Critical/Casual
+
+In-person or at-home: where can you do the job?
+
+## Behind the scenes
+
+These are jobs that can be done without being at an event.
+
+### Pizza quartermaster - Ned
+
+On the day of an event, note how many people have RSVP’d yes, use the Beauty’s Pizza web site to order enough pizza in an interesting mix, pay with the Boston Python debit card.
+
+### Twitter-meister - Srini
+
+Run the @bostonpython Twitter account. Follow interesting people, tweet things. Announce events and sponsors.
+
+Time: 10min/day; On-going; Can be shared; Casual; At-home.
+
+### Sponsor wrangler - Brian
+
+Find pizza sponsors for events. Contact potential sponsors about how sponsorship works. Answer their questions. Arrange sponsors for specific events. Keep meetup.com sponsor list updated. Send thank you emails, be really grateful.
+
+### Jobs detective
+
+Search job postings for Python jobs in the Boston-area. Collect names and email addresses of people we can contact about sponsoring.
+
+Time: 1 hour/week; On-going; Can be shared; Casual; At-home.
+
+### Hosting wrangler - Mike
+
+Find companies to host our events. Reserve Microsoft NERD when it's available. Communicate with potential hosts about requirements. Evaluate potential spaces in person. Confirm dates for events, add to meetup.com.
+
+### Meadhall reserver - Ned
```

Comment:
We should generalize from Meadhall.   Also, throughout, I'm not sure how to notate that some jobs simply aren't being done a lot, or at all at the moment.  Maybe we don't need to?